pages/common/fossa.md

```diff
@@ -0,0 +1,24 @@
+# fossa
+
+> CLI for the Fossa service - Generate realtime license audits, vulnerability scans and reports about dependencies licenses.
```

Comment:
This is fine for now, but I think for next time this might have been better split over 2 description lines.